workers/unity/Packages/com.improbable.gdk.core/Worker/ConnectionHandlers/MultithreadedSpatialOSConnectionHandler/MultiThreadedSpatialOSConnectionHandler.cs

```diff
@@ -51,6 +51,16 @@ public void PushMessagesToSend(MessagesToSend messages)
             serializationHandler.EnqueueMessagesToSend(messages);
         }
 
+        public string GetWorkerFlag(string name)
```

Comment:
This should not be on here. Add it to the diff and the view

Reply:
Should probably only go on the view in this case. I.e. - diff = flag changes, view = current flags

Reply:
Agreed

Reply:
Yup that's what I did 😉 

`View` has a local dictionary which is updated with the changes in the `ViewDiff`.